Makefile

```diff
@@ -1,8 +1,11 @@
-.PHONY: default all requirements configure ironic ocp_run install_config clean ocp_cleanup ironic_cleanup host_cleanup cache_cleanup registry_cleanup workingdir_cleanup podman_cleanup bell assisted_deployment_cleanup
-default: requirements configure build_installer ironic ocp_run bell
+.PHONY: default all requirements configure ironic ocp_run install_config clean ocp_cleanup ironic_cleanup host_cleanup cache_cleanup registry_cleanup workingdir_cleanup podman_cleanup bell
+default: requirements configure build_installer ironic install_config ocp_run bell
```

Comment:
This is fixing a rebase mistake in a previous commit on this branch, it'd be better to fix that commit instead of doing it here?